src/benchmark/ips.cr

```diff
@@ -100,7 +100,7 @@ module Benchmark
           ips = measurements.map { |m| item.cycles.to_f / m.total_seconds }
           item.calculate_stats(ips)
 
-          item.bytes_per_op = (bytes.to_f / cycles.to_f).round.to_i
+          item.bytes_per_op = (bytes.to_f / cycles.to_f).round.to_u64
```

Comment:
Could we also change this to:

```suggestion
          item.bytes_per_op = (bytes / cycles).round.to_u64
```

since `/` already does float division

Reply:
Actually, nevermind. Not related to this fix.